### PR TITLE
filled in readme

### LINKED ISSUE #3: Fill in README
A solid `README` is a great way to make a positive first impression on a person interested in your project (and sharpen your [Markdown](https://help.github.com/articles/github-flavored-markdown/) skills). Take some time to fill in the existing `README` with details about the project.

Below are a few examples I pulled to help guide the structure. Feel free to dig into their underlying Markdown if you see a formatting technique that is unfamiliar.

(Don't worry about understanding any of the technical stuff in these. Mostly their structure and intent is what's important.)

**Examples**
- https://github.com/confluentinc/bottledwater-pg

A big emphasis on making it easy for an interested party to test things out and honesty about the status of the project.
- https://github.com/codahale/sneaker

Starts off with honesty, then lots of details around setup and configuration. Follows with a nice architecture diagram, and since it is a security focused project, ends with potential attack vectors.
- https://github.com/jmhodges/justrun

Long description about what the tool was meant to do. Follows with examples, usage, and a comparison against other similar tools.

/cc @cahdeemer

### PR DESCRIPTION
**Pull request by cahdeemer**
Hi, Matt and Hector!

I hope I did this right!  I filled in my readMe on a new branch (called readme) and pushed it up on my fork.  I think my next step is a pull request.  Is that right?  

Let me know if I messed something up or need to change anything.  Thanks so much!

Christina

Resolves #3

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,4 +3,40 @@ A map app for beginners in the OSMP 2015 program.
...
+The practice of mapping grant funds is not new or uncommon.  Many foundations, particularly those with a global reach, use maps to communicate impact or illustrate areas of interest.
+
+[The Foundation Center's Foundations Map tool] (http://maps.foundationcenter.org/home.php) is comprehensive and powerful.  It is accessible only via subscription, but some foundations make their Foundation Center map available on their websites.
+
+Examples of Foundation Center maps include:
+
+ *  [Charles Stewart Mott Foundation] (http://www.mott.org1/grantsandguidelines/ExploreGrants/maps)
+ * [Zellerbach Family Foundation] (http://zff.org/grantmaking/)
+
+Other examples (not using Founations Map tool):
```
Comment (mmcfarland) [inline README.md]: Typo `Founations` -> `Foundations`

**Comment by mmcfarland** [comment]
This looks like great documentation, nice job.  A cool trick that GitHub provides for is referencing issues within a pull request, then closing it automatically when it gets merged.  
Check out:
https://help.github.com/articles/closing-issues-via-commit-messages/

You can edit your PR message to include something like "Closes #3" to get that behavior.

File: README.md [hunk 2]
```diff
@@ -3,4 +3,40 @@ A map app for beginners in the OSMP 2015 program.
...
+[The Foundation Center's Foundations Map tool] (http://maps.foundationcenter.org/home.php) is comprehensive and powerful.  It is accessible only via subscription, but some foundations make their Foundation Center map available on their websites.
+
+Examples of Foundation Center maps include:
+
+ *  [Charles Stewart Mott Foundation] (http://www.mott.org1/grantsandguidelines/ExploreGrants/maps)
+ * [Zellerbach Family Foundation] (http://zff.org/grantmaking/)
+
+Other examples (not using Founations Map tool): 
+
+ *  [FFA Environmental Grants] (http://www.communitywalk.com/map/1743255)
```
Comment (hectcastro) [inline README.md:30]: Nice use of bullets.

**Comment by hectcastro** [comment]
Great job making use of Markdown's features in your `README`.

It may not be immediately obvious, but to fix the typo Matt found, fix it and create a new commit with just the typo fix. Once that's committed, push your local repository up to your fork and the new commit should show up in this pull request.

The steps might look something like:

``` bash
$ git branch
  master
* readme

# Fix your typo

$ git commit -m "Fixed typo"
$ git push origin readme
```

**Comment by cahdeemer** [comment]
So much awesome learning today!  I fixed the typo and followed Hector's instructions for pushing the revision back up - I did have to do "git add README.md", though, because git said that the file wasn't staged for the commit.  

I also edited my PR message with the reference to the issue.  Had no idea that was possible.  No clue.  

Also, thanks for responding so quickly.  :smile:

**Comment by hectcastro** [comment]
:+1: This looks good to me. Merging...

**Pull request merged**